masonry/src/contexts.rs

```diff
@@ -362,13 +362,13 @@ impl_context_method!(EventCtx<'_>, {
     /// Set the cursor icon.
     ///
     /// This setting will be retained until [`clear_cursor`] is called, but it will only take
-    /// effect when this widget is [`hot`] and/or [`has_pointer_capture`]. If a child widget also
+    /// effect when this widget is [`hovered`] and/or [`has_pointer_capture`]. If a child widget also
```

Comment:
I understand that it sort of reads poorly, but I think this link should be `is_hovered` since that's the actual name. If it is just "hovered", then it probably should lose the backticks to avoid letting people think it is a function name.